src/cloud-api-adaptor/test/e2e/assessment_runner.go

```diff
@@ -276,15 +276,15 @@ func (tc *TestCase) Run() {
 						t.Logf("Expected Pod State: %v", tc.podState)
 						yamlData, err := yaml.Marshal(pod.Status)
 						if err != nil {
-							fmt.Println("Error marshaling pod.Status to YAML: ", err.Error())
+							t.Logf("Error marshaling pod.Status to YAML: ", err.Error())
```

Comment:
needs fixing ^

Reply:
Also the commit message is a little ugly.